ldms/src/sampler/lustre_client/lustre_client_general.c

```diff
@@ -212,21 +252,31 @@ static int llite_stats_sample(const char *stats_path,
                             str1, &val1, &val2);
                 if (rc == 2) {
```

Comment:
I think it would be simpler to understand if we start, in sscanf, to catch the units inside the square brackets. Then we can just make new condition that is basically:

`if units == "usecs" and count == 4`

(4 because catchings the units string will add another field. The other numbers will need to be bumped too.)


Granted, the numbers change once we start parsing the units string.